packages/@sanity/desk-tool/src/lib/useUnique.ts

```diff
@@ -0,0 +1,12 @@
+import {isEqual} from 'lodash'
+import {useRef} from 'react'
+
+export function useUnique<ValueType>(value: ValueType): ValueType {
```

Comment:
could a better name for this be useDeepUnique to better reflect that it does a (potentially expensive) deep comparison on every render? I guess it's also worth adding a note about when *not* to use - I can easily imagine cases where rendering is cheap and using this will actually degrade performance.